### ISSUE TITLE
Drag Tabs to New Window

### ISSUE THREAD

**Issue by benkaiser** [Eugeny/terminus#581]
Dragging a tab out of the window does not create a new window with that terminal in it for me. I would expect it to behave similarly to chrome/firefox tabs where dragging out a tab creates a new window with that tab.

System: Windows 10
Version: Alpha 65

**Comment by audioeptesicus** [comment]
+1

**Comment by Bitzo** [comment]
+1

**Comment by mquinnv** [comment]
no doubt, this is my biggest pain point with terminus right now. other than this terminus is excellent!

**Comment by SergioSuarezDev** [comment]
+1

**Comment by alimoezzi** [comment]
+1

**Comment by mrjcleaver** [comment]
ChatGPT: compare terminus to tabby

🔹 Choose Terminus if you want: Detached windows or multi-pane workflows

🔍 Key Note:
Both are developed by the same author, Eugeny. In fact:

🧠 Terminus was the predecessor to Tabby. Tabby is essentially the next-generation rethinking of Terminus, with better performance and long-term support.

But if you're deciding between them today, Tabby is the more actively developed and recommended path forward — unless you specifically need detached windows or Terminus-only plugins.

It goes on to suggest other apps if multiwindow support is key to one's workflow.

**Comment by Eugeny** [comment]
AI slop goes crazy

**Comment by mrjcleaver** [comment]
Is it wrong then? Did Terminus never support multiwindow?

**Comment by Eugeny** [comment]
No, it was never supported. And I literally renamed it overnight to avoid potential copyright issues with _Termius_.

Implementing draggable tabs is extremely hard as Tabby windows are separate processes and the entire state of a tab is contained within the renderer process.

**Comment by skull-squadron** [comment]
Do hard things. Hard things are valuable. 

Another thing to consider is to decouple the underlying terminal process from the UI process so that the UI can be upgraded or crash and restarted without disconnecting anything. iTerm on (other guy's platform) does this.

**Comment by Eugeny** [comment]
> Do hard things. Hard things are valuable. 

That's some wild shit to post demanding free work from someone.

**Comment by FrankOVD** [comment]
+1

- @skull-squadron 's suggestion could be something to consider
-  We are on GitHub, we could also try and help
- The README.md file contains a [ko-fi link](https://ko-fi.com/J3J8KWTF)
